README.md

```diff
@@ -4,11 +4,32 @@
 
 NGINX Kubernetes Gateway is an open-source project that provides an implementation of the [Gateway API](https://gateway-api.sigs.k8s.io/) using [NGINX](https://nginx.org/) as the data plane. The goal of this project is to implement the core Gateway APIs -- `Gateway`, `GatewayClass`, `HTTPRoute`, `TCPRoute`, `TLSRoute`, and `UDPRoute` -- to configure an HTTP or TCP/UDP load balancer, reverse-proxy, or API gateway for applications running on Kubernetes. NGINX Kubernetes Gateway is currently under development and supports a subset of the Gateway API.
 
-> Warning: This project is actively in development (pre-alpha feature state) and should not be deployed in a production environment.
+> Warning: This project is actively in development (beta feature state) and should not be deployed in a production environment.
 > All APIs, SDKs, designs, and packages are subject to change.
 
-## Run NGINX Kubernetes Gateway
+## Getting Started
 
-1. [Build](docs/building-the-image.md) the NGINX Kubernetes Gateway container image.
-2. [Install](docs/installation.md) NGINX Kubernetes Gateway.
-3. Deploy various [examples](examples). 
+1. [Quick Start on a kind cluster](docs/running-on-kind.md).
+2. [Build](docs/building-the-image.md) the NGINX Kubernetes Gateway container image.
+3. [Install](docs/installation.md) NGINX Kubernetes Gateway.
+4. Deploy various [examples](examples). 
+
```

Comment:
I wonder how we can help people to ensure they don't use main branch docs and examples for a release like v0.1.0. One way to do it is something similar to https://github.com/nginxinc/kubernetes-ingress/blob/v1.5.0/README.md#nginx-ingress-controller-releases .  However, perhaps we can have something simpler, like a note here.

Reply:
I like being consistent with NIC whenever possible. I'll add a table for releases and link the examples, docs, and manifests there.

Reply:
Updated, let me know what you think.

I'm a little unsure about the release descriptions.

Reply:
looks good! 👍